modin/core/storage_formats/base/query_compiler.py

```diff
@@ -3021,6 +3021,68 @@ def groupby_size(
             result.columns = result.columns[:-1].append(pandas.Index(["size"]))
         return result
 
+    @doc_utils.add_refer_to("GroupBy.rolling")
+    def groupby_rolling(
```

Comment:
do we need to reflect this anyhow in docs, change supported api spreadsheet maybe?

Reply:
It seems that there's no mentioning of any inner methods of `groupby` (and so no `groupby.rolling`) in the supported api spreadsheet. There's just a line saying that `pandas.DataFrame.groupby` is supported and that's it ([src](https://github.com/modin-project/modin/blob/master/docs/supported_apis/dataframe_supported.rst)).